### DISCUSSION TITLE
Authentication header is printed in logs when workflowConfig.loggerLevel = "DEBUG"

### DISCUSSION THREAD

**Discussion by giacomochiarella** [open-metadata/OpenMetadata#10643] [Q&A]
The authentication header is printed out when the debug mode is on when using openmetadata-ingestion==0.13.2.0 in python. Is there a way to enable debug mode without printing it out?

**Comment by harshach** [comment]
@giacomochiarella I think we want to show the auth header in debug mode to make sure the token is correct. Do you see this as potential issue? we can disable it but it is debug mode, most other services log all these details to help the user to debug.
Let us know

**Comment by giacomochiarella** [comment]
@harshach when the token is wrong I will not even connect to OM. I get an error about the connection. It would be useful to have the debug mode not printing it because om jobs usually take long, I have to remember to delete logs in cloudwatch, a bit frustrating
